index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="style.css">
+<link rel="stylesheet" href="css/font-awesome.min.css">
+<title>Tim's Portfolio</title>
+</head>
+<body>
+  <div id="nav-bar" class="nav-bar">
+      <h1 id="home-nav">Home</h1>
+      <h1 id="work-nav">Projects</h1>
+      <h1 id="about-nav">About Me</h1>
+    </div>
+    </div>
+    </div>
+  </div>
+  <div id="home" class="home">
+    <h1>Home</h1>
+    <a href="https://github.com/tmitchell92"><i class="fa fa-github fa-5x" class="github" aria-hidden="true"></i></a>
+  </div>
+  <section id="projects">
+      <article class="template" data-category="cat">
+    <h1>Projects</h1>
+    <div class="byline">
```

Comment:
This div is unnecessary.  The project should be contained in the article tag with only the a + h2 and a p tag containing your description.